archetype/maven-url-handler/src/main/java/io/helidon/build/archetype/maven/url/mvn/Handler.java

```diff
@@ -24,11 +24,16 @@
 /**
  * Maven Url Stream Handler.
  */
-public class MavenHandler extends URLStreamHandler {
+public class Handler extends URLStreamHandler {
```

Comment:
`Handler` is too generic of a name, can you make it more domain specific ? (E.g. `MavenURLHandler`)